packages/components/src/components/contract-card/contract-card-items/contract-card-header.jsx

```diff
@@ -111,6 +111,7 @@ ContractCardHeader.propTypes = {
     onClickSell: PropTypes.func,
     server_time: PropTypes.object,
     id: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+    is_sold: PropTypes.bool,
```

Comment:
to fix typescript error in **summary card** component:
<img width="1120" alt="Screenshot 2023-03-22 at 12 38 58" src="https://user-images.githubusercontent.com/103181650/226863720-6a331589-e02e-48f9-bc45-8c7a61e7d486.png">